### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by hshiah**
Fix empty reminder bug and add reminder and goal to command summary

### CODE REVIEW

**Review by YFshadaow — APPROVED** [review]
LGTM

**Review by ryan1604 — APPROVED** [review]
LGTM

File: src/main/java/seedu/financialplanner/commands/AddReminderCommand.java [hunk 1]
```diff
@@ -1,22 +1,46 @@
...
+
+        String dateString = rawCommand.extraArgs.get("d");
+        if(dateString.isEmpty()){
+            throw new IllegalArgumentException("Reminder date cannot be empty");
+        }
+
+        try {
+            date = LocalDate.parse(dateString, formatter);
+        } catch (DateTimeParseException e) {
+            throw new IllegalArgumentException("Reminder date must be in the format yyyy-MM-dd");
```
Comment (ryan1604) [inline src/main/java/seedu/financialplanner/commands/AddReminderCommand.java]: Can change format to follow cashflow eg. dd/MM/yyyy

File: src/main/java/seedu/financialplanner/reminder/Reminder.java [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+    public Reminder(String type, LocalDate date) {
         this.type = type;
         this.date = date;
     }
 
     public Reminder(String type, String date, String status) {
         this.type = type;
-        this.date = date;
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+        this.date = LocalDate.parse(date, formatter);
```
Comment (ryan1604) [inline src/main/java/seedu/financialplanner/reminder/Reminder.java:19]: format for here too

**Review by ryan1604 — APPROVED** [review]
LGTM, make sure to add safeguards for loading reminder

File: src/main/java/seedu/financialplanner/commands/AddReminderCommand.java [hunk 3]
```diff
@@ -1,22 +1,46 @@
...
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
 public class AddReminderCommand extends Command {
     private final String type;
-    private final String date;
+    private final LocalDate date;
 
     public AddReminderCommand(RawCommand rawCommand) throws IllegalArgumentException {
         String typeString = String.join(" ", rawCommand.args);
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
```
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/commands/AddReminderCommand.java]: possible to change this to dd/MM/yyyy to standardise dates

**Review by NeoMinWei — APPROVED** [review]
LGTM, just change the date format to standardise it across the program

**Pull request merged**